vulnerabilities/tests/__init__.py

```diff
@@ -3,6 +3,53 @@
 # VulnerableCode is a trademark of nexB Inc.
 # SPDX-License-Identifier: Apache-2.0
 # See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
-# See https://github.com/nexB/vulnerablecode for support or download.
+# See https://github.com/aboutcode-org/vulnerablecode for support or download.
 # See https://aboutcode.org for more information about nexB OSS projects.
 #
+
+from django.utils import timezone
```

Comment:
Getting these "fixtures" in the __init__ looks a little bit weird to me. Why not move these directly in the modules you use them? Also it is OK to have these duplicated in each module IMHO as it may help with understanding the code and avoid dependencies between tests?

Reply:
Done, moved fixtures to individual test modules.